channels/ON-CHAIN.md

```diff
@@ -124,13 +148,18 @@ the withdrawn tokens arrive.
 Serialization defined [here](../serializations.md#channel-withdraw-transaction)
 
 
-- `channel_id`: channel id as recorded on chain
+- `channel_id`: channel id as recorded on-chain
 - `to`: receiver of the withdraw
 - `amount`: amount of tokens withdrawn
 - `ttl`:
 - `fee`:
+- `state_hash`: the root hash of the channel state tree after the withdraw had been applied
```

Comment:
See [analogous point on channel_deposit on whether consensus requires this state_hash to be distinct from the one in the channel object in the block state tree](https://github.com/aeternity/protocol/pull/119/files?utf8=%E2%9C%93&diff=split&w=1#r196072046).